Modules/Registration/Common/test/itkMultiResolutionImageRegistrationMethodTest.cxx

```diff
@@ -135,7 +135,7 @@ itkMultiResolutionImageRegistrationMethodTest(int, char *[])
   registration->SetInitialTransformParameters(initialParameters);
   ITK_TEST_SET_GET_VALUE(initialParameters, registration->GetInitialTransformParameters());
 
-  ParametersType initialTransformParametersOfNextLevel(1);
+  ParametersType initialTransformParametersOfNextLevel = ParametersType(1);
```

Comment:
Which problem did you encounter with the original code, `ParametersType initialTransformParametersOfNextLevel(1)`?

Reply:
As mentioned in the commit message, Valgrind detected that the values were not initialized.

Reply:
I'm sorry, but I still don't see a _semantic_ difference between the original:

    ParametersType initialTransformParametersOfNextLevel(1);

and the proposed:

    ParametersType initialTransformParametersOfNextLevel = ParametersType(1);


Both lines of code call the `explicit OptimizerParameters(SizeValueType dimension)` constructor, which does _not_ initialize its elements.

I think you might fix the Valgrind issue by using the `explicit OptimizerParameters(const SizeValueType dimension, const ValueType & value)` constructor instead. For example:

    ParametersType initialTransformParametersOfNextLevel(1, 2.0);

If you do so, the initial parameter value will be `2.0`.

Reply:
You're right. Overlook from my side. Fixed in 2942c8d.